plugins/flexibleengine/src/main/java/org/eclipse/xpanse/plugins/flexibleengine/monitor/utils/FlexibleEngineMetricsService.java

```diff
@@ -339,9 +339,16 @@ private Map<MonitorResourceType, MetricInfoList> getTargetMetricsMap(
                             monitorResourceType.toValue(), deployResource.getResourceId());
                 }
             }
-        }
-        return targetMetricsMap;
+            return targetMetricsMap;
 
+        } else {
+            log.error(
+                    "No monitor metrics available for the service, "
+                            + "Please wait for 3-5 minutes and try again.");
+            throw new ClientApiCallFailedException(
+                    "No monitor metrics available for the  service, "
```

Comment:
looks like there is an extra space before the word "service". Same in both places.